GameServer.cpp

```diff
@@ -460,13 +460,10 @@ void GameServer::DoTimedUpdates()
 	LogCounters(counters);
 }
 
-
-int GameServer::FindGameInfoClient(sockaddr_in &from, unsigned int clientRandValue)
+std::size_t GameServer::FindGameInfoClient(sockaddr_in &from, unsigned int clientRandValue)

```

Comment:
The logic in functions FindGameInfoClient and FindGameInfoServer should be combined in a future refactor.

Reply:
I think you're right.

Actually, if we used standard library containers, they might reduce down to a standard library container call, with a lambda for the search predicate.